src/popup/components/AccountCardWrapper.vue

```diff
@@ -0,0 +1,79 @@
+<template>
```

Comment:
This component actually is not a wrapper as it doesn't wrap anything. It's the card itself.